sdk/openai/ai-openai/samples-dev/openAi.ts

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * Demonstrates how to get completions for the provided prompt using OpenAI API.
```

Comment:
nit: since the sampe API is used from difference resources, maybe 
```suggestion
 * Demonstrates how to get completions for the provided prompt using OpenAI hosted service.
```
or something similar?